python/tests/transformers/tf_image_test.py

```diff
@@ -176,3 +176,37 @@ def test_prediction_vs_tensorflow_inceptionV3(self):
         self.compareClassSets(tf_topK, transformer_topK)
         self.compareClassOrderings(tf_topK, transformer_topK)
         self.compareArrays(tf_values, transformer_values, decimal=5)
+
+    def test_prediction_vs_tensorflow_inceptionV3_sql(self):
+        output_col = "my_prediction"
+        image_df = image_utils.getSampleImageDF()
+
+        # An example of how a pre-trained keras model can be used with TFImageTransformer
+        with KSessionWrap() as (sess, g):
+            with g.as_default():
+                K.set_learning_phase(0)    # this is important but it's on the user to call it.
+                # nChannels needed for input_tensor in the InceptionV3 call below
+                image_string = utils.imageInputPlaceholder(nChannels=3)
+                resized_images = tf.image.resize_images(image_string,
+                                                        InceptionV3Constants.INPUT_SHAPE)
+                # keras expects array in RGB order, we get it from image schema in BGR =>
+                # need to flip
+                preprocessed = preprocess_input(imageIO._reverseChannels(resized_images))
+                model = InceptionV3(input_tensor=preprocessed, weights="imagenet")
+                graph = tfx.strip_and_freeze_until([model.output], g, sess, return_graph=True)
+
+        transformer = TFImageTransformer(channelOrder='BGR', inputCol="image", outputCol=output_col, graph=graph,
+                                         inputTensor=image_string, outputTensor=model.output,
+                                         outputMode="sql",
+                                         outputMapping={model.output.name: 'my_prediction'})
+        transformed_df = transformer.transform(image_df.limit(10))
```

Comment:
I saw a limit here but no limit with `tf_values`. We don't have more than 10 images. But it is better to remove this limit or use the same limit in TensorFlow.